week2/homework/src/help.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+{
```

Comment:
what's the purpose of this bracket ?

Reply:
Hi @ Sherinex, Thanks for your feedback.

As I have learned from previous lectures, the purpose of these brackets is to create a local scope that prevents any conflicts with other files... Which is not the case in this app.
I will remove them.

Reply:
this is true for javascript files when sent to browser but for node, it co es with a module system, that does scoping (per module basis)